src/main/resources/static/vulnerableApp.js

```diff
@@ -70,75 +70,86 @@ function _callbackForInnerMasterOnClickEvent(
   };
 }
 
-function handleFirstElementAutoSelection(vulnerableAppEndPointData) {
-  if (vulnerableAppEndPointData.length > 0) {
-    vulnerabilitySelected = vulnerableAppEndPointData[0]["Name"];
-    detailTitle.innerHTML = vulnerableAppEndPointData[0]["Description"];
-    let isFirst = true;
-    for (let key in vulnerableAppEndPointData[0]["Detailed Information"]) {
-      let column = document.createElement("div");
-      column.id = "0." + key;
-      let textNode = document.createTextNode(
-        vulnerableAppEndPointData[0]["Detailed Information"][key]["Level"]
-      );
-      column.appendChild(textNode);
-      column.className = "inner-master-item";
-      column.addEventListener(
-        "click",
-        _callbackForInnerMasterOnClickEvent(
-          vulnerableAppEndPointData,
-          0,
-          key,
-          vulnerabilitySelected
-        )
-      );
-      if (isFirst) {
-        column.click();
-        isFirst = false;
-      }
-      innerMaster.appendChild(column);
+function getVariant(detailedInformation) {
+  return detailedInformation["Variant"] === "SECURE";
+}
+
+function getSvgVariant(isSecure) {
```

Comment:
we can rename it to `_getSvgElementForVariant` or some other name which depicts that it is returning the javascript element. "_" is to depict that it is a private utility.

Reply:
Resolved in 422b8d3